browser/search_engines/android/java/strings/android_search_engines_strings.grd

```diff
@@ -0,0 +1,185 @@
+    <file path="translations/android_search_engines_strings_mn.xtb" lang="mn" />
+    <file path="translations/android_search_engines_strings_ml.xtb" lang="ml" />
+    <file path="translations/android_search_engines_strings_mr.xtb" lang="mr" />
+    <file path="translations/android_search_engines_strings_ms.xtb" lang="ms" />
+    <file path="translations/android_search_engines_strings_my.xtb" lang="my" />
+    <file path="translations/android_search_engines_strings_nl.xtb" lang="nl" />
+    <file path="translations/android_search_engines_strings_no.xtb" lang="no" />
+    <file path="translations/android_search_engines_strings_pl.xtb" lang="pl" />
+    <file path="translations/android_search_engines_strings_pt-BR.xtb" lang="pt-BR" />
+    <file path="translations/android_search_engines_strings_pt-PT.xtb" lang="pt-PT" />
+    <file path="translations/android_search_engines_strings_ro.xtb" lang="ro" />
+    <file path="translations/android_search_engines_strings_ru.xtb" lang="ru" />
+    <file path="translations/android_search_engines_strings_si.xtb" lang="si" />
+    <file path="translations/android_search_engines_strings_sk.xtb" lang="sk" />
+    <file path="translations/android_search_engines_strings_sl.xtb" lang="sl" />
+    <file path="translations/android_search_engines_strings_sq.xtb" lang="sq" />
+    <file path="translations/android_search_engines_strings_sr.xtb" lang="sr" />
+    <file path="translations/android_search_engines_strings_sr-Latn.xtb" lang="sr-Latn" />
+    <file path="translations/android_search_engines_strings_sv.xtb" lang="sv" />
+    <file path="translations/android_search_engines_strings_sw.xtb" lang="sw" />
+    <file path="translations/android_search_engines_strings_ta.xtb" lang="ta" />
+    <file path="translations/android_search_engines_strings_te.xtb" lang="te" />
+    <file path="translations/android_search_engines_strings_th.xtb" lang="th" />
+    <file path="translations/android_search_engines_strings_tr.xtb" lang="tr" />
+    <file path="translations/android_search_engines_strings_uk.xtb" lang="uk" />
+    <file path="translations/android_search_engines_strings_uz.xtb" lang="uz" />
+    <file path="translations/android_search_engines_strings_vi.xtb" lang="vi" />
+    <file path="translations/android_search_engines_strings_zh-CN.xtb" lang="zh-CN" />
+    <file path="translations/android_search_engines_strings_zh-TW.xtb" lang="zh-TW" />
+  </translations>
+  <release seq="1">
+    <messages fallback_to_english="true">
+    <!-- Start of Search engines strings -->
+    <message name="IDS_YOUTUBE" desc="Text for youtube">
+        YouTube
+      </message>
+    <message name="IDS_QUICK_SEARCH_ENGINES" desc="Title for quick search engines option">
+        Quick search engines
+      </message>
+      <message name="IDS_QUICK_SEARCH_ENGINES_WITH_HYPHEN" desc="Title for quick search engines with hyphen">
+        Quick-Search Engines
+      </message>
```

Comment:
Just curious, what's the distinction between the title with hyphen and the one without?

Reply:
we use hyphened version for the menu in settings and we use the one without it as a title of the page when we show the UI.